From e14ea63454aa8e1cdea30a804336f81c57facb80 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Thu, 24 Aug 2023 12:38:01 -0700
Subject: [PATCH 74/76] UBUNTU: SAUCE: apparmor4.0.0 [74/76]: advertise
 disconnected.path is available

While disconnected.path has been available for a while it was never
properly advertised as a feature. Fix this so that userspace doesn't
need special casing to handle it.

Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 security/apparmor/apparmorfs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/security/apparmor/apparmorfs.c b/security/apparmor/apparmorfs.c
index f1e9da932d64..341f46be6b7f 100644
--- a/security/apparmor/apparmorfs.c
+++ b/security/apparmor/apparmorfs.c
@@ -2600,6 +2600,7 @@ static struct aa_sfs_entry aa_sfs_entry_unconfined[] = {
 
 static struct aa_sfs_entry aa_sfs_entry_profile[] = {
 	AA_SFS_FILE_BOOLEAN("interruptible",		1),
+	AA_SFS_FILE_BOOLEAN("disconnected.path",	1),
 	{ }
 };
 
-- 
2.42.0

